team.go

```diff
@@ -70,6 +70,8 @@ type OrganizationAccess struct {
 	ManageRunTasks        bool `jsonapi:"attr,manage-run-tasks"`
 	// **Note: This field is still in BETA and subject to change.**
 	ManageProjects bool `jsonapi:"attr,manage-projects"`
+	ReadWorkspaces        bool `jsonapi:"attr,read-workspaces"`
```

Comment:
This needs a gofmt run! merging from upstream is always an adventure, huh

Reply:
Ah, sorry about that! TIL!